contract-ui/tabs/overview/cards/custom-contract-code.tsx

```diff
@@ -398,6 +416,19 @@ const InteractiveAbiFunction: React.FC<InteractiveAbiFunctionProps> = ({
           </>
         )}
 
+        {abiFunction?.stateMutability === "payable" && (
+          <>
+            <Divider borderColor="borderColor" />
+            <FormControl gap={0.5}>
+              <FormLabel>Native Token Value</FormLabel>
+              <Input {...register(`value`)} />
+              <FormHelperText>
+                The native currency value to send with this transaction.
```

Comment:
should probably add something like `ex: 0.01 to send 0.01 ether` so its clear that we do the parsing for you